Questie.lua

```diff
@@ -623,10 +638,10 @@ local options = {
 					name = "Reset Questie",
 				},
 				Spacer_B = _QuestieOptions:Spacer(12),
-				github_text = {
+				reset_text = {
```

Comment:
Is this change intended for this pull request?

Reply:
Yes this change is intended as it's a copy/paste error from the original commit.  The table key for "GitHub_text" is already used below so it was ts that texnot showing this text.  by changing it to "reset_text" it let show and not be overwritten.